src/components/AssigneeSelector.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div>
+    <select v-model="assignee" @change="onChange">
+      <option value="null">No Assignee</option>
+      <option
+        v-for="user in users"
+        :key="user.gid"
+        :value="user.gid"
+      >{{ user.name }}</option>
+    </select>
+  </div>
+</template>
+
+<script lang="ts">
+import store from "@/store";
+import { defineComponent, onMounted, ref } from "vue";
+import { createNamespacedHelpers } from "vuex";
+const { mapState } = createNamespacedHelpers("asana");
+
+export default defineComponent({
+  computed: {
+    ...mapState(["users"]),
+  },
+  setup() {
+    const assignee = ref<string | null>();
+
+    onMounted(() => {
+      assignee.value = store.state["preferences"].taskEditorSectionIdAndTask.task.assignee?.gid ?? null;
+    })
+
+    const onChange = (event: Event) => {
+      const assignee = (event.target as HTMLSelectElement).value;
```

Comment:
Thinking through this, I think we want to break this up. In this case, you have "two things" that are referncing the value of the assignee; the piece in the v-model, and this event. Generally, you only want one, because if someone else modifies this code, they may not realize there are two. And when you have two things, it is very easily to fall into "thing1 updates things2 updates thing1 ...". This code isn't there, but we want to do what we can now to prevent it.

So, for this, you have two main options:
- Remove the event param from your onChange function and just use assignee.value
- Remove the change event completely, and add a watch to assignee. This is likely the preferred option because it is easier to see things "cascade" from the v-model